.github/workflows/check-dist-sha256.yml

```diff
@@ -0,0 +1,51 @@
+name: Check dist sha256
```

Comment:
I decided instead of us manually checking this, why not automate it?

So decided to write a workflow to solve this issue. 

The example repo had this :)

Reply:
Hey, unfortunately reusable workflows can't be called from other reusable workflows so it's not of much use as we'll want to use it in our builder/generator reusable workflows.
https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations

Reply:
If this is meant as a pre-submit that runs on PRs then I suggest changing the file name to start with 'pre-submit.' so that it matches our other checks.

Reply:
I have changed the file name.